cluster-provision/base/scripts/dnsmasq.sh

```diff
@@ -6,15 +6,17 @@ NUM_NODES=${NUM_NODES-1}
 NUM_SECONDARY_NICS=${NUM_SECONDARY_NICS-0}
 
 ip link add br0 type bridge
+echo 0 > /proc/sys/net/ipv6/conf/br0/disable_ipv6
 ip link set dev br0 up
 ip addr add dev br0 192.168.66.02/24
+ip -6 addr add 2001:0db8:0:f101::1 dev br0
 
 for i in $(seq 1 ${NUM_NODES}); do
   n="$(printf "%02d" ${i})"
   ip tuntap add dev tap${n} mode tap user $(whoami)
   ip link set tap${n} master br0
   ip link set dev tap${n} up
-  DHCP_HOSTS="${DHCP_HOSTS} --dhcp-host=52:55:00:d1:55:${n},192.168.66.1${n},node${n},infinite"
+  DHCP_HOSTS="${DHCP_HOSTS} --dhcp-host=52:55:00:d1:55:${n},192.168.66.1${n},[2001:0db8:0:f101::1${n}],node${n},infinite"
```

Comment:
2001:0db8: is used for doc examples https://en.wikipedia.org/wiki/Reserved_IP_addresses.

I think we should use fd00::/8 for local addressing.

Reply:
done.